src/components/application_manager/test/commands/hmi/hmi_notifications/hmi_notifications_test.cc

```diff
@@ -851,7 +851,7 @@ TEST_F(HMICommandsNotificationsTest,
 
 TEST_F(HMICommandsNotificationsTest,
        OnDeviceStateChangedNotificationDeviceStateUnpaired) {
-  std::string device_id = "device_id";
+  const std::string device_id = "AA15F2204D6B";
```

Comment:
@anosach-luxoft , it is just a random MAC adress for test. It must contain 12 symbols. The previous contained 9 symbols and that is why test fails.